src/itwinai/loggers.py

```diff
@@ -707,10 +707,8 @@ def create_logger_context(self, rank: Optional[int] = None) -> None:
         if not self.should_log():
             return
 
-        os.makedirs(os.path.join(self.savedir, "wandb"), exist_ok=True)
-        self.active_run = wandb.init(
-            dir=os.path.abspath(self.savedir), project=self.project_name
-        )
+        os.makedirs(self.savedir / "wandb", exist_ok=True)
```

Comment:
In this case you can actually do 

```python
(self.savedir / "wandb").mkdir(exist_ok=True)
```
and avoid `os` altogether.

Reply:
cool!